bdanmaku.lua

```diff
@@ -30,7 +31,8 @@ function download_xml()
 		mp.msg.debug('no XML danmaku found')
 		return
 	end
-	xml_filename = TMPDIR..'/'..os.time()..'.danmaku.xml'
+	os.execute('mkdir -p '..TMPDIR)
```

Comment:
Good suggestion, but it doesn't work on Windows.

Reply:
Use `powershell mkdir` to make tmpdir for Windows, when linux `mkdir` is failed. Fix in [c5a6e2f](https://github.com/UlyssesZh/bdanmaku/pull/1/commits/c5a6e2ff19afc3dab58b52649d5590117cbe85d4)

Reply:
Alternatively, use **lua-filesystem** for more OS versatility, but that lua-filesystem library is required.